psi-ms.obo

```diff
@@ -59,6 +68,18 @@ id: has_structured_representation_in_format
 name: has structured representation in format
 def: "The term is encoded in the structure of the named file format rather than as an explicit parameter, and should not be used as one in that format." []
 
+[Typedef]
+id: has_value_concept
+name: has_value_concept
+
+[Typedef]
+id: has_column
+name: has_column
+
+[Typedef]
+id: has_optional_column
+name: has_optional_column
+
```

Comment:
The typedef for `has_metric_category` is missing

Reply:
added in b7f9ca66a11fd8bbe779e4ec665d538529a84873